internal/operands/common-templates/reconcile.go

```diff
@@ -87,6 +95,32 @@ func (c *commonTemplates) Cleanup(request *common.Request) error {
 	return nil
 }
 
+func pauseCRs(request *common.Request) {
+	patch := []byte(`{"metadata":{"annotations":{"kubevirt.io/operator.paused": "true"}}}`)
+	var kubevirtCommonTemplatesBundles kvsspv1.KubevirtCommonTemplatesBundleList
+	err := request.Client.List(context.TODO(), &kubevirtCommonTemplatesBundles, &client.ListOptions{})
+	if err != nil {
+		request.Logger.Error(err, fmt.Sprintf("Error listing common template bundles: %s", err))
+		return
+	}
+	if err == nil && len(kubevirtCommonTemplatesBundles.Items) > 0 {
+		for _, kubevirtCommonTemplatesBundle := range kubevirtCommonTemplatesBundles.Items {
+			err = request.Client.Patch(context.TODO(), &kvsspv1.KubevirtCommonTemplatesBundle{
```

Comment:
`request` has a `Context`, would be better to use it

Reply:
Why have you inlined the CR here instead of providing `&kubevirtCommonTemplatesBundle` or `&kubevirtCommonTemplatesBundles.Items[i]` to the function?